### PR TITLE
New `backup` command

### PR DESCRIPTION
**Pull request by afbora**

### CODE REVIEW

File: commands/backup.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		// execution commands list
+		$commands = [
+			// navigates to the target directory to ignore parent folders in zip file
+			'cd ' . escapeshellarg($targetPath) . ';',
+			// set backup file path
+			'zip -r ' . escapeshellarg($backupFile),
+			// set target backup directory
+			escapeshellarg(empty($relativeTargetPath) === true ? '*' : ('./' . $relativeTargetPath . '/*')),
+			// exclude backup directory
+			'-x ' . escapeshellarg('backup/*')
```
Comment (distantnative) [inline commands/backup.php:43]: Wouldn't this also exclude any e.g. content folder named `backup` or plugin named `backup` etc?

**Comment by distantnative** [comment]
I am not sure I understand the purpose of the `root` argument. Like if someone just wants to backup blueprints?

**Comment by afbora** [comment]
> I am not sure I understand the purpose of the `root` argument. Like if someone just wants to backup blueprints?

Yes. For ex, backup only `content` or `site` or `accounts` folder.

File: commands/backup.php (hunk 1 above)
Comment (afbora) [reply commands/backup.php:43]: Hmm, maybe. I need to check 🙂

**Comment by distantnative** [comment]
I think that should be reflected in the backup name then. Otherwise it might be hard to know which file is a full backup, which just a part etc.

**Review by bastianallgeier — APPROVED** [review]
I made a few small adjustments again, but overall this is also really cool.

**Pull request merged**